dateconv/dateconv.go

```diff
@@ -134,6 +134,32 @@ func TotalDaysInBSYear(year int) (int, error) {
 	return sum, nil
 }
 
+// now returns the current time
+var now = func() time.Time {
+	return time.Now()
+}
+
+// TotalDaysSpanned returns the total number of days spanned in the
+// current year inclusive of the current day.
+func TotalDaysSpanned() (int, error) {
+	bsDate := ToBS(now())
+	days, ok := bsDaysInMonthsByYear[bsDate.year]
+
+	if !ok {
+		return -1, fmt.Errorf("Year should be in between %d and %d", bsLBound, bsUBound)
+	}
+
+	sum := 0
```

Comment:
it seems `sum` could be initialized to `bsDate.days`

Reply:
Yes. I wanted to be more explicit. 😄